api/CONFIG.md

```diff
@@ -47,6 +47,7 @@ headless | false |  If true, the window will never be displayed.
 name |  |  The name of the program and executable to be output. Can't contain spaces or special characters. Required field.
 output | "build" |  The binary output path. It's recommended to add this path to .gitignore.
 script |  |  The build script. It runs before the `[build] copy` phase.
+pass.build.arguments | false |  If true, it will pass build arguments to the build script. WARNING: this could be deprecated in the future.
```

Comment:
This name is discussable. I'm open to another opinions.

@jwerle I made the separator a `.` here. Is it ok or should I revert to `_` for the `0.5` release?

Reply:
The `.` is correct. We should probably call this property `build.script.forward_arguments`